apps/modernization-ui/src/apps/patient/add/extended/asExtendedNewPatientEntry.spec.ts

```diff
@@ -0,0 +1,186 @@
+import { today } from 'date';
+import { NewPatientEntry } from '../NewPatientEntry';
+import { PatientNameCodedValues } from 'apps/patient/profile/names/usePatientNameCodedValues';
+import { LocationCodedValues, StateCodedValue } from 'location';
+import { CodedValue } from 'coded';
+import { Selectable } from 'options';
+import { act, renderHook } from '@testing-library/react-hooks';
+import { asExtendedNewPatientEntry } from './asExtendedNewPatientEntry';
+import { Deceased, Gender, Suffix } from 'generated/graphql/schema';
+
+const nameCodes: PatientNameCodedValues = {
+    types: [
+        {
+            value: 'T',
+            name: 'Test'
+        }
+    ],
+    prefixes: [{ value: 'SR', name: 'Senior' }],
+    suffixes: [{ value: 'JR', name: 'Junior' }],
+    degrees: []
+};
+const addressCodes: LocationCodedValues = {
+    countries: [{ value: 'USA', name: 'United states america' }],
+    states: {
+        all: [{ value: 'AR', name: 'Arkansas', abbreviation: 'ARK' }],
+        byAbbreviation: jest.fn(),
+        byValue: jest.fn()
+    },
+    counties: {
+        byState: jest.fn()
+    }
+};
+const raceCode: Selectable[] = [{ value: 'A', name: 'Asian' }];
+
+describe('Basic form to extended form data transfer', () => {
+    it('basic name to extended', async () => {
+        const date = today();
+        const initial: NewPatientEntry = {
+            asOf: date,
+            firstName: 'test first',
+            identification: [],
+            phoneNumbers: [],
+            emailAddresses: [],
+            lastName: 'test last',
+            middleName: 'test middle',
+            suffix: Suffix.Jr
+        };
+
+        const { result } = renderHook(() => asExtendedNewPatientEntry(initial, nameCodes, addressCodes, raceCode));
```

Comment:
issue: `asExtendedNewPatientEntry` is not a hook, you should be able to call it directly without the `renderHook` function.

```typescript
const actual = asExtendedNewPatientEntry(initial, nameCodes, addressCodes, raceCode);
```